pimd/pim_cmd.c

```diff
@@ -6861,12 +6870,33 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
-DEFUN (ip_pim_v6_secondary,
-       ip_pim_v6_secondary_cmd,
-       "ip pim send-v6-secondary",
-       IP_STR
-       "pim multicast routing\n"
-       "Send v6 secondary addresses\n")
+DEFPY(igmp_group_watermark, igmp_group_watermark_cmd,
+      "ip igmp watermark-warn (10-60000)$limit",
+      IP_STR IGMP_STR
+      "Configure group limit for watermark warning\n"
+      "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	pim->igmp_watermark_limit = limit;
+
+	return CMD_SUCCESS;
+}
+
+DEFPY(no_igmp_group_watermark, no_igmp_group_watermark_cmd,
+      "[no] ip igmp watermark-warn",
+      NO_STR IP_STR IGMP_STR
+      "Unconfigure watermark warning limit for igmp groups\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	pim->igmp_watermark_limit = 0;
+
+	return CMD_SUCCESS;
+}
+
+DEFUN(ip_pim_v6_secondary, ip_pim_v6_secondary_cmd, "ip pim send-v6-secondary",
```

Comment:
Why is this code changed?

Reply:
I haven't changed it. Diff is showing this way.

Reply:
it sure is. please take a second and look at your diff.  this defun has each parameter on it's own line in the original code and now it doesn't.

Reply:
Yes got my mistake. Will change it.

Reply:
undone the changes.